content/en/mobile/_index.md

```diff
@@ -80,11 +80,30 @@ On the Incidents page, you can view, search and filter all incidents that you ha
 
 ## Widgets
 
-### Monitor saved views
+### SLOs widget
+
+{{< img src="mobile/slo_widget.png" alt="widget page" responsive="true" style="width:100%; background:none; border:none; box-shadow:none;">}}
+
+View your [SLOs][13] from your mobile home screen with Datadog widgets. You can add any SLOs from your organization as a widget, along with a timeframe.
+
+Timeframe options are:
+- 7 days
+- 30 days
+- 90 days
+- Previous week
+- Previous month
+- Week to date
+- Month to date
+
+You can also choose which dashboard from your organization opens in the app when you tap on an SLO widget, allowing you to quickly dig deeper into your metrics.
```

Comment:
```suggestion
You can also choose a dashboard to open when you tap on an SLO widget, allowing you to quickly dig deeper into your metrics.
```

Reply:
updating copy to suggested change but adding a few clarifying words